src/utils/TwoColumns.tsx

```diff
@@ -1,12 +1,16 @@
 import React, { FC } from 'react';
 
-interface Props {
+interface InterfaceTwoColumnsProps {
   columnOne: React.ReactNode;
-  columnTwo: React.ReactNode | JSX.Element;
+  columnTwo: React.ReactNode;
   reverse?: boolean;
 }
 
-const TwoColumns: FC<Props> = ({ columnOne, columnTwo, reverse = false }) => {
+const TwoColumns: FC<InterfaceTwoColumnsProps> = ({
+  columnOne,
+  columnTwo,
+  reverse = false,
```

Comment:
Why was this file edited? You have changed the functionality of the home page. That is beyond the scope of the issue. Revert.